Apps/Laboratory/src/Delegates/MockLaboratoryDelegate.cs

```diff
@@ -26,6 +27,7 @@ namespace HealthGateway.Laboratory.Delegates
     /// <summary>
     /// Implementation that uses HTTP to retrieve laboratory information.
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
Can't we test this?

Reply:
@sslaws is it just a Mock...Delegate? But we can try to find a way to test it if needed.

Reply:
+1, since is mock I dont think is necessary to test it